packages/flutter_tools/templates/app_shared/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -1,11 +1,18 @@
+# Project-level configuration.
 cmake_minimum_required(VERSION 3.10)
 project(runner LANGUAGES CXX)
 
+# The name of the executable created for the application. Change this to change
+# the on-disk name of your application.
 set(BINARY_NAME "{{projectName}}")
+# The unique GTK application identifier for this application.
```

Comment:
I wonder if it's worth referencing the relevant GTK docs. One issue is that I can't actually _find_ the GTK docs for this other than a bunch of references in Gnome docs. Gnome is built on GTK but I'm not sure whether or not they impose any additional requirements on application IDs that GTK does not. :/

https://developer-old.gnome.org/ChooseApplicationID/